bindings/wasm/lib/bindings.ts

```diff
@@ -9,7 +9,8 @@ import { __UtilsMethods__ } from './utils';
 // Import needs to be in a single line, otherwise it breaks
 // prettier-ignore
 // @ts-ignore: path is set to match runtime transpiled js path when bundled.
-import { initLogger, createClient, destroyClient, createSecretManager, createWallet, callClientMethod, callSecretManagerMethod, callUtilsMethodRust, callWalletMethod, destroyWallet, listenWallet, getClientFromWallet, getSecretManagerFromWallet, listenMqtt, migrateStrongholdSnapshotV2ToV3 } from '../wasm/iota_sdk_wasm';
+import { initLogger, getClient, createClient, destroyClient, createSecretManager, createWallet, callClientMethod, callSecretManagerMethod, callUtilsMethodRust, callWalletMethod, destroyWallet, listenWallet, getSecretManagerFromWallet, listenMqtt, migrateStrongholdSnapshotV2ToV3 } from '../wasm/iota_sdk_wasm';
+import { SecretManagerMethodHandler, WalletMethodHandler } from '.';
```

Comment:
'SecretManagerMethodHandler' is defined but never used
'WalletMethodHandler' is defined but never used